packages/paste-core/components/inline-control-group/src/InlineControlGroup.tsx

```diff
@@ -30,11 +30,13 @@ const InlineControlGroup = React.forwardRef<HTMLFieldSetElement, InlineControlGr
       orientation = 'vertical',
       required,
       i18nRequiredLabel,
-      fieldStyleProps, // Only used in Visual Picker
+      fieldStyleProps,
       ...props
     },
     ref
   ) => {
+    const isVisualPicker = Boolean(fieldStyleProps); // This prop is only used in Visual Picker
```

Comment:
suggestion: prefer an existence check like `fieldStyleProps != null` instead of a type coercion